spring-cloud-dataflow-docs/src/main/asciidoc/appendix-howto.adoc

```diff
@@ -75,162 +71,154 @@ SPRING_APPLICATION_JSON='{
 $ cf restage dataflow-server
 ----
-+
 
-(_where, `reactor.ipc.netty` is the global package for everything `reactor-netty` related_)
+(where `reactor.ipc.netty` is the global package for everything `reactor-netty` related.)
+
+NOTE: Similar to the `local-deployer` and `cloudfoundry-deployer` options as discussed above, there are equivalent settings available for Apache YARN, Apache Mesos, and Kubernetes variants.
+See the respective link:https://github.com/spring-cloud?utf8=%E2%9C%93&q=spring-cloud-deployer[SPI implementations] for more detail about the packages to configure for logging.
 
-NOTE: Similar to the `local-deployer` and `cloudfoundry-deployer` options as discussed above, there
-are equivalent settings available for Apache YARN, Apache Mesos and Kubernetes variants, too. Check out the
-respective link:https://github.com/spring-cloud?utf8=%E2%9C%93&q=spring-cloud-deployer[SPI implementations] to
-find out more details about the packages to configure for logging.
 
-=== Application Logs
 
-The streaming applications in Spring Cloud Data Flow are Spring Boot applications and they can be
-independently setup with logging configurations.
+[[troubleshooting-application-logs]]
+=== Application Logs
 
-For instance, if you'd have to troubleshoot the `header` and `payload` specifics that are being passed
-around source, processor and sink channels, you'd be deploying the stream with the following
-options.
+The streaming applications in Spring Cloud Data Flow are Spring Boot applications, and they can be independently setup with logging configurations.
 
+For instance, if you must troubleshoot the `header` and `payload` specifics that are being passed around source, processor, and sink channels, you should deploy the stream with the following options:
 
 [source,bash]
 ----
 dataflow:>stream create foo --definition "http --logging.level.org.springframework.integration=DEBUG | transform --logging.level.org.springframework.integration=DEBUG | log --logging.level.org.springframework.integration=DEBUG" --deploy
 ----
 
-(_where, `org.springframework.integration` is the global package for everything Spring Integration related,
-which is responsible for messaging channels_)
+(where `org.springframework.integration` is the global package for everything Spring Integration related,
+which is responsible for messaging channels.)
 
-These properties can also be specified via `deployment` properties when deploying the stream.
+These properties can also be specified with `deployment` properties when deploying the stream, as follows:
 
 [source,bash]
 ----
 dataflow:>stream deploy foo --properties "app.*.logging.level.org.springframework.integration=DEBUG"
 ----
 
-==== Remote Debugging 
+==== Remote Debugging
 
-Data Flow allows you to debug the deployed applications.
-This is accomplished using enabling the remote debugging feature of the JVM through deployment properites.
+Data Flow lets you debug the deployed applications.
```

Comment:
"The Local-deployer  implementation of Data Flow lets you debug the deployed applications"